updatecli/weekly.d/jenkinscontroller-agents-packer-agent-all-in-one.yaml

```diff
@@ -0,0 +1,72 @@
+---
+name: Bump Packer Agent Templates (all-in-one) Version
+
+scms:
+  default:
+    kind: github
+    spec:
+      user: "{{ .github.user }}"
+      email: "{{ .github.email }}"
+      owner: "{{ .github.owner }}"
+      repository: "{{ .github.repository }}"
+      token: "{{ requiredEnv .github.token }}"
+      username: "{{ .github.username }}"
+      branch: "{{ .github.branch }}"
+
+sources:
+  packerImageVersion:
+    kind: githubrelease
+    spec:
+      owner: "jenkins-infra"
+      repository: "packer-images"
+      token: "{{ requiredEnv .github.token }}"
+      username: "{{ .github.username }}"
+
+  getLatestInboundAllInOneContainerImageX86:
+    kind: dockerdigest
+    name: Get digest of the jenkinsciinfra/jenkins-agent-ubuntu-22.04 image
+    dependson:
+      - packerImageVersion
+    spec:
+      image: "jenkinsciinfra/jenkins-agent-ubuntu-22.04"
+      tag: '{{ source "packerImageVersion"}}'
+      architecture: amd64
+
+  getLatestInboundAllInOneContainerImageARM:
+      kind: dockerdigest
+      name: Get digest of the jenkinsciinfra/jenkins-agent-ubuntu-22.04 image
+      dependson:
+        - packerImageVersion
+      spec:
+        image: "jenkinsciinfra/jenkins-agent-ubuntu-22.04"
+        tag: '{{ source "packerImageVersion"}}'
+        architecture: arm64
+
+conditions:
+  checkAllInOneContainerImages:
+    disablesourceinput: true
+    name: Check that x86 and arm64 all-in-one images are different
+    kind: shell
+    spec:
+      command: test {{ source "getLatestInboundAllInOneContainerImageX86" }} != {{ source "getLatestInboundAllInOneContainerImageARM" }}
+
+targets:

```

Comment:
(blocker) There is a target missing here (this one:https://github.com/jenkins-infra/jenkins-infra/blob/07babac855913f2b3ccd0a8070c1e1b87db95fb9/updatecli/weekly.d/jenkinscontroller-agents.yaml#L79C3-L86).

Rationale: we want this manifest to update *both* the Azure gallery image AND the Docker digest of what we call the "all in one image", because  both comes from the same release on jenkins-infra/packer-images (same content!). It's their distribution format which differs, but we want the update to be synced.